dev/issues.rst

```diff
@@ -16,38 +16,58 @@ Examples include "Duplicate of #123", "Discovered to be configuration
 error", "Rendered moot by #123" and so on. We don't use the "invalid" or
 "wontfix" labels.
 
+api
+    Issues related to the API and configuration.
+
 bug
-    The issue is a verified bug.
+    Issues verified to be a bug.
 
 build
-    The issue is caused by or requires changes to the build system
-    (scripts or Docker image).
+    Issues caused by or requiring changes to the build system (scripts
+    or Docker image).
+
+dependencies
+    Pull requests that update a dependency file.
+
+documentation
+    Issues related to the documentation.
 
 enhancement
-    This is a new feature or an improvement of some kind, as
-    opposed to a problem (bug).
+    New features or improvements of some kind, as opposed to a problem
+    (bug).
 
 frozen-due-to-age
-    This is set automatically on issues when they have been closed untouched
-    for a long time, together with the issue being locked against
-    discussion.
+    Set automatically on issues when they have been closed untouched
+    for a long time, together with being locked for discussion.
+
+good first issue
+    Good starting points for new contributors, contained in scope and
+    size, and clear about what is the desired outcome.
 
-good-first-issue
-    These issues are good starting points for new contributors. They are
-    contained in scope and size, and it should be clear what is the desired
-    outcome.
+needs-triage
+    New issues needed to be validated.
 
 not-our-bug
     Rare and temporary label used only when we want to keep an issue
     open for visibility, but the real problem is somewhere else.
 
+pr-merged
+    Legacy label used in the past for pull requests merged to the main
+    tree.
+
 protocol
-    This requires a change to the protocol.
+    Issues that require a change to the protocol.
+
+skip-changelog
+    Set on bugs that have never been in a released stable version, i.e.
```

Comment:
it's set on PRs I think

Reply:
I have followed https://github.com/syncthing/docs/issues/559#issuecomment-671472241
>And skip-changelog replaced unreleased, i.e. unreleased can go. And skip-changelog is mainly used in the scenario you described on unreleased.

and https://github.com/syncthing/docs/issues/559#issuecomment-671795761:
>And skip-changelog should only be used for bugs introduced in an RC, which don't need to be mentioned in the change log because they weren't present in any previous version

What is the answer then? There have been only five PRs marked with this label, all created last year.

The description itself has literally been taken from the now removed `unreleased` label. I have only updated the Syncthing version numbers in the example.

Reply:
You're right, it seems we set it on both, perhaps we should state bugs/PRs as it's clearly either.

Reply:
It has defined meaning when set on bugs, as it prevents those bugs from then being mentioned in the changelog. It is meaningless to the automation when set on PRs, as PRs don't go into the changelog at all by themselves. I think the description here is fine, if someone wants to label the corresponding PR like this then it does no harm.